src/styles.js

```diff
@@ -8,13 +8,15 @@ export const GlobalStyle = createGlobalStyle`
 	}
 
 	:root{
-		--bgColor: #F6F6F6;
+		--offWhite: #F6F6F6;
+		--bgColor: var(--offWhite);
```

Comment:
do you need both 
`--offwhite and --bgColor`? because they have the same value :)

Reply:
later